source/en/sdr.rst

```diff
@@ -465,39 +465,39 @@ Interpreting Results
 
 .. note:: The resolution of the output rasters will be the same as the resolution of the DEM provided as input.
 
-.. note:: The raster results of SDR are given as values *per pixel*. To convert the per pixel values to per hectare values, you will adjust by the size of your pixels relative to one hectare. For example: If *1 pixel = 900 m2*, then the conversion from tons/pixel to tons/ha would be: *(tons/pixel x pixel/900 m2 x 10000 m2/ha)* or *(the per pixel value x (10000/900))*. The number will get bigger when the pixel is smaller than a hectare.
+.. note:: The raster results of SDR are given as values *per hectare*. To convert the per hectare values to per pixel values, adjust by the size of your pixels relative to one hectare. For example: If *1 pixel = 900 m2*, then the conversion from metric tons per hectare (Mg/ha) to metric tons per pixel (Mg/px) would be: *(Mg/ha x 1/10000 ha/m2 x 900 m2/px)* or *(the per hectare value x (900/10000))*. The number will get smaller when the pixel size is smaller than a hectare.
 
 * **[Workspace]** folder:
 
     * **Parameter log**: Each time the model is run, a text (.txt) file will be created in the Workspace. This file will list the parameter values and output messages for that run and will be named according to the service, the date and time, and the suffix. When contacting NatCap about errors in a model run, please include the parameter log.
 
-    * **rkls.tif** (type: raster; units: tons/pixel/year): Total potential soil loss per pixel in the original land cover from the RKLS equation. Equivalent to the soil loss for bare soil. (Eq. :eq:`usle`, without applying the :math:`C` or :math:`P` factors)
+    * **rkls.tif** (type: raster; units: metric tons/hectare/year): Total potential soil loss in the original land cover from the RKLS equation. Equivalent to the soil loss for bare soil. (Eq. :eq:`usle`, without applying the :math:`C` or :math:`P` factors)
 
-    * **sed_export.tif** (type: raster; units: tons/pixel/year): The total amount of sediment exported from each pixel that reaches the stream. (Eq. :eq:`e_i`)
+    * **sed_export.tif** (type: raster; units: metric tons/hectare/year): The total amount of sediment exported from each pixel that reaches the stream. (Eq. :eq:`e_i`)
 
-    * **sed_deposition.tif** (type: raster; units: tons/pixel/year): The total amount of sediment deposited on the pixel from upslope sources as a result of trapping. (Eq. :eq:`ti`)
+    * **sed_deposition.tif** (type: raster; units: metric tons/hectare/year): The total amount of sediment deposited on the pixel from upslope sources as a result of trapping. (Eq. :eq:`ti`)

```

Comment:
```suggestion
    * **sed_deposition.tif** (type: raster; units: t/(ha*year)): The total amount of sediment deposited on the pixel from upslope sources as a result of trapping. (Eq. :eq:`ti`)

```

Reply:
My usual inclination is to spell things out as much as possible, and consistently. So personally I'd be more in favor of "metric tons/(hectare*year)" (or "metric tons/hectare/year", which reads a little more clearly to me, but let's pick one way of writing it and do it consistently throughout the UG).  If I was a beginner with the models, I'm not sure I'd know what "t" stands for.